romancal/jump/jump_step.py

```diff
@@ -49,10 +49,10 @@ def process(self, input):
 
             # Extract the needed info from the Roman Data Model
             meta = input_model.meta
-            r_data = input_model.data
+            r_data = input_model.data.value
             r_gdq = input_model.groupdq
             r_pdq = input_model.pixeldq
-            r_err = input_model.err
+            r_err = input_model.err.value
```

Comment:
It would be good form in these two lines to do something like r_data = input_model.data.to(u.DN).value, essentially making sure that the unit is the one you expect before taking the .value.  For other cases with units this is more important (e.g., you don't want to treat meters and kilometers the same here); here it would be more of a consistency check.

Reply:
I like this, but want to spin it off into another issue to do throughout the code.

Reply:
Put it in this issue #644